content/en/blog/signal-boosting.md

```diff
@@ -48,8 +48,9 @@ By themselves signals are not terribly interesting until combined with the two o
 
 _Computed signals_ derive new values from other signals using _compute functions_.
 
-```ts
-import { signal, computed } from "@preact/signals-core";
+```js
+// --repl
+import { signal } from "@preact/signals";
```

Comment:
I'm not so sure that we need REPL links for all of these, as the comments and text are meant to walk one through the examples.

Also this particular REPL wouldn't work, `computed` is in fact needed.

Reply:
Fixed imports. 
I somehow copied the original import in post "signals-core" and wasn't paying attention.  Sorry about that.
REPL helped me at least, only did the sections that can run; a couple continuation samples left alone.
If you want me to take the REPL refs out, I can.